src/pages/PageUsingComponents.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { SplitClient, SplitTreatments } from '@splitsoftware/splitio-react';
+import { feature_1, feature_2, feature_3 } from '../sdkConfig';
+
+/* This example showcasts SplitClient and SplitTreatments components */
```

Comment:
change "showcasts" for "showcases" or "show". (googled showcasts meaning and it takes me to showcases meaning)